development/compiling/compiling_for_x11.rst

```diff
@@ -120,7 +120,7 @@ Building export templates
 .. warning:: Linux binaries usually won't run on distributions that are
              older than the distribution they were built on. If you wish to
              distribute binaries that work on most distributions,
-             you should build them on an old distribution such as Ubuntu 16.04.
+             you should build them on an old distribution such as Ubuntu 18.04.
```

Comment:
Maybe just say "you should build them on Ubuntu 18.04"

Reply:
Other old distributions such as CentOS can be a valid choice, but I agree Ubuntu LTS is the most relevant choice when it comes to building portable games.

Reply:
My comment was specifically referring to it as "an old distribution"

Reply:
I think it's fine to start writing documentation like this. That statement will become true shortly; in about 2 months, 20.04 LTS will make 18.04 LTS an old Ubuntu LTS.